src/main/kotlin/sc/iview/commands/view/StartRecordingVideo.kt

```diff
@@ -54,15 +55,21 @@ class StartRecordingVideo : Command {
     private lateinit var videoEncodingQuality // listed as an enum here, cant access from java https://github.com/scenerygraphics/scenery/blob/1a451c2864e5a48e47622d9313fe1681e47d7958/src/main/kotlin/graphics/scenery/utils/H264Encoder.kt#L65
             : String
 
-    @Parameter
+    @Parameter(label="Use hardware encoding?")
     private var useHWEncoding = true
     // Note that this parameter is affirmative, while scenery's is a negation. We flip this later
 
+    @Parameter(style="save")
+    private lateinit var filename: File
```

Comment:
Maybe put the filename parameter first, followed by the two boolean toggles? Feels a bit strange to have the file chooser sandwiched in between.